spec/sidekiq/form526_submission_failure_email_job_spec.rb

```diff
@@ -13,7 +21,7 @@
     Sidekiq::Job.clear_all
     allow(VaNotify::Service)
       .to receive(:new)
-      .with(Settings.vanotify.services.benefits_disability.api_key)
+      .with(Settings.vanotify.services.benefits_disability.api_key, callback_metadata)
```

Comment:
Hmm I might make sure to run this end to end if you can locally if you haven't. I assume you did and looking at [the constructor sig](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/va_notify/lib/va_notify/service.rb#L19) I think this will work. 

Just thinking out loud as I've definitely bitten myself by mocking a call like this and finding out something buried in the service I'm using didn't do what I expected 🤣 .

Reply:
This was actually just me changing the existing tests to pass... but I can investigate and potentially add more tests

Reply:
Yeah makes sense, this isn't behind a flipper in prod anymore so some small risk in this breaking the mailers but we would catch that in the metrics and address it then, so not the end of the world if that happens and we could always resend the mailers